wiki/Tournaments/GBC/GBC_2023_Spring/en.md

```diff
@@ -0,0 +1,207 @@
+---
+tags:
+  - GBC
+  - GBC2023
+  - GBC 2023
+  - GBC 2023 Spring
+---
+
+# GB Cup 2023 Spring
+
+![GBC 2023 Spring Banner](img/GBC-2023-Spring-banner.png) 
+
+The **GB Cup 2023 Spring** (***GBC 2023 Spring***) is an osu!mania 4K point-based weekly group tournament hosted by Team GB. It is the third instalment of the GB Cup.
+
+## Tournament schedule
+
+| Event | Timestamp (UTC+08) |
+| --: | :-- |
+| Registration phase | 2023-03-04/2023-03-18 (23:59) |
+| Regular stage 1 | 2023-03-25/2023-04-02 |
+| Regular stage 2 | 2023-04-08/2023-04-16 |
+| Regular stage 3 | 2023-04-22/2022-04-30 |
+| Regular stage 4 | 2023-05-06/2022-05-14 |
+| Regular match result showcase | 2023-05-15 (20:00) |
+| Playoff stage week 1 | 2023-05-20/2023-05-21 |
+| Playoff stage week 2 | 2023-05-27/2023-05-28 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "playoff stage 1st place") | Profile badge (pending), KanonBot profile badge, 100 RMB |
+| ![Silver crown](/wiki/shared/crown-silver.png "playoff stage 2nd place") | KanonBot profile badge, 90 RMB |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "playoff stage 3rd place") | KanonBot profile badge, 70 RMB |
+| *4th place* | 50 RMB |
+| *5th place – 6th place* | 40 RMB |
+| *7th place – 8th place* | 30 RMB |
+| *9th place – 12th place* | 20 RMB |
+| *13th place – 16th place* | 10 RMB |
+| *All participants* | KanonBot profile badge (all groups), 15 RMB (Group B), 25 RMB (Group A), 35 RMB (Group S), 45 RMB (Group EX) |
```

Comment:
what is kanonbot?

Reply:
KanonBot is a chatbot that provides querying osu player profile/scores, which is very high usage for the Chinese player community. When using KanonBot, you can query the player's basic information (PP, Acc, Rank, etc.), and KanonBot has a similar system to osu badge, which can display the badge that the player has on KanonBot, and our reward is this kind of badge.

Reply:
may be worth having a link to it, then

Reply:
i guess you asked once on GBC 2022 Autumn page xD

Reply:
apparently i did lol https://github.com/ppy/osu-wiki/pull/8705#discussion_r1089702990